### PR TITLE
Adding className as a common prop for all widgets

### PR DESCRIPTION
**Pull request by divdavem**

### CODE REVIEW

File: core/lib/commonProps.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface AllWidgetsCommonPropsAndState {
```
Comment (fbasso) [inline core/lib/commonProps.ts:1]: I think that just `WidgetsCommonPropsAndState` as a name is enough ;)

Comment (divdavem) [reply core/lib/commonProps.ts]: Thank you for your suggestion. I have renamed the interface.

**Review by fbasso — APPROVED** [review]
Fine for me, thanks @divdavem !

**Comment by divdavem** [comment]
@fbasso Thank you for your review!

**Pull request merged**